libs/design/src/atoms/form/form-field/README.md

```diff
@@ -1,8 +1,15 @@
 # Form field
-Form field is a wrapping component for form control elements.
+Form field is a wrapping component that provides consistent styling and behavior for form control elements.
 
 ## Overview
-It's used to style certain controls that would otherwise be impossible to style with normal css and organize error messages alongside their associated form controls.
+It's used to style certain controls that would otherwise be impossible to style with normal css and organize labels, hints, and error messages alongside their associated form controls.
+
+The following Daffodil Design components are designed to work inside a `<daff-form-field>`:
+
+- DaffInputComponent (<input daff-input />)
```

Comment:
Will these link automatically? I think we should omit the selectors.

Reply:
No, I updated to link to them to the usage docs.